multiraft/multiraft.go

```diff
@@ -957,11 +997,12 @@ func (s *state) createGroup(groupID roachpb.RangeID, replicaID roachpb.ReplicaID
 		}
 		if replica.StoreID == s.storeID {
 			log.Infof("node %s campaigning because initial confstate is %v", s.nodeID, cs.Nodes)
-			if err := s.multiNode.Campaign(context.Background(), uint64(groupID)); err != nil {
+			if err := raftGroup.Campaign(); err != nil {
 				return err
 			}
 		}
 	}
+	log.Infof("node %v creating group %v", s.nodeID, groupID)
```

Comment:
Either remove this line or make it `log.V(3)` (but we already have a `log.V(3)` at the start of this method so I don't think we need it)